tests/ocp/1-005_validate_route_tls/03-secret-tls-data.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example
+spec:
+  server:
+    route:
+      enabled: true
+      tls:
+        externalCertificate:
```

Comment:
Can you add a test case for validating the error message when both `tls.externalCertificate` and `tls.certificate` is set ?

Reply:
I will create a follow up PR for deprecation events and logs, will address this with it.